__tests__/app.test.js

```diff
@@ -96,3 +97,36 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("should respond with a 200 status code when successful", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  it("should when successful respond with an array of all article objects without the body property present", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
```

Comment:
Great. However, if you are using a forEach, you need to test the length of the array to avoid false positives for an empty array.